docs/documentation/overview.md

```diff
@@ -2,27 +2,27 @@
 
 Zulip has three major documentation systems:
 
-- Developer and sysadmin documentation: Documentation for people
+- **Developer and sysadmin documentation**: Documentation for people
   actually interacting with the Zulip codebase (either by developing
   it or installing it), and written in Markdown.
 
-- Core website documentation: Complete webpages for complex topics,
+- **Core website documentation**: Complete webpages for complex topics,
```

Comment:
Maybe we should rebrand this as "Product landing pages"?  I feel like "landing pages" is sorta a more typical name for what we mean by this sort of fancier page.

@alya thoughts?  Possibly you should just plan to do a pass on this paragraph after we merge this PR, since it's orthogonal to what's being changed here.

Reply:
Do the landing pages even fall under "documentation"? It seems strange to categorize them as such, but I guess a contributor might update, e.g., the integrations page, which is why these pages are included here?

Anyway, yeah, let's bookmark that as a separate edit pass.

Reply:
I think some of the more detailed ones like /features kinda fall into that category, but yeah, another option would be to delete this paragraph or replace it with just a brief mention that there's other user-facing content in landing pages and the like.

Reply:
I'm probably the exception, but I think I did end up interacting some with the core website documentation for my first PR with Zulip (which is still pending and perhaps wasn't the best choice for a first issue) as I updated some text on [this page of the Zulip site](https://zulip.com/apps/), so some sort of mention might be worth while. I don't think I really registered it as different from the other documentation pages at the time.